### DISCUSSION TITLE
about _gray_aug in augmentation.py

### DISCUSSION THREAD

**Discussion by laisimiao** [STVIR/pysot#405] [General]
in  [augmentation.py line65](https://github.com/STVIR/pysot/blob/master/pysot/datasets/augmentation.py#L65), there is _gray_aug method:
```
def _gray_aug(self, image):
    grayed = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
    image = cv2.cvtColor(grayed, cv2.COLOR_GRAY2BGR)
    return image
```
It seems it doesn't do the gray transformation?

**Comment by ZhiyuanChen** [comment]
Why do you think it doesn't do the gray transformation?

**Comment by laisimiao** [comment]
Oh,  firstly I thought it's just an  inversed operation, but after I look through the [API](https://docs.opencv.org/2.4/modules/imgproc/doc/miscellaneous_transformations.html#cvtcolor):
![image](https://user-images.githubusercontent.com/37405764/89876629-7ed96880-dbf1-11ea-800b-a5cf545733b2.png)
and by a small experiment I found it do does the gray transfoemation!
